### ISSUE TITLE
`no-empty-label`: Add quotes to error report

### ISSUE THREAD

**Issue by IanVS** [eslint/eslint#3526]
Currently, the report of `no-empty-label` does not put quotes around the label being flagged:

```js
/*eslint no-empty-label: 2*/

labeled:
var x = 10;
```

```
3:1  error  Unexpected label labeled  no-empty-label
```

I think this would be a little more readable as:

```
3:1  error  Unexpected label "labeled"  no-empty-label
```

**Comment by mysticatea** [comment]
:+1: 

`no-unused-vars` has the same issue.
